### ISSUE TITLE
Normal if nan appear in target?

### ISSUE THREAD

**Issue by fgdfgfthgr-fox** [janelia-cellmap/cellmap-segmentation-challenge#162]
I notice sometimes (although rarely) the target array returned by the dataloader will contain nans. Is that supposed to happen?
The class is ecs (and nothing else) and both the input and target resolution is 32nm.

**Comment by rhoadesScholar** [comment]
Yes! Actually, this is 100% intended - it makes masking out unknown regions very easy during loss calculations. For instance the provided [CellMapLossWrapper](https://github.com/janelia-cellmap/cellmap-segmentation-challenge/blob/main/src/cellmap_segmentation_challenge/utils/loss.py) takes care of this for you, if you supply it with any PyTorch loss function.

Why this is handy:
- Example 1: We sample the edge of a provided GT array, but want our image to be taken from a cube at 45 degrees to the original grid. This would put a corner outside of the known GT annotations. By filling that corner with NaN we can throw it out during loss calculations.

- Example 2: On our secret menu, so to speak, we have included functionality that lets you use known mutually exclusive classes to serve as true-negative masks for unannotated classes. For instance, if a volume has nuclei annotations, we can know that there are no mitochondria within those nuclei locations, but we don't know if there might be mitochondria elsewhere - so we mask the rest of the locations with NaNs.
